benchmarks/from_csv.py

```diff
@@ -0,0 +1,124 @@
+import glob
+import os
+from datetime import datetime
+
+import vaex
+
+
+def test_open_two_big_csv_convert():
```

Comment:
These functions will be converted to use ASV in #620

Reply:
No, i'm ok having this in now